src/cmn/Makefile

```diff
@@ -1,3 +1,19 @@
+#
+# Copyright (c) 2019, Infosys Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
/*
 * Copyright 2019-present Open Networking Foundation
 *
 * SPDX-License-Identifier: Apache-2.0
 *
 */

WE need copyright like this..Put your company name instead of ONF. Do the same thing all the places.

Reply:
Updated the copyrights.